src/workerd/jsg/observer.h

```diff
@@ -143,6 +143,8 @@ struct InternalExceptionObserver {
     bool isInternal;
     bool isFromRemote;
     bool isDurableObjectReset;
+    using InternalErrorId = kj::FixedArray<char, 24>;
```

Comment:
Can we pull the typedef out of util.c++ so we have one source of truth with the magic number?

I imagine JSG's dependency graph might be too crazy to make that feasible, though.